common/lib/container-definitions/src/runtime.ts

```diff
@@ -111,7 +111,7 @@ export interface IContainerContext extends IDisposable {
     readonly configuration: IFluidConfiguration;
     readonly clientId: string | undefined;
     readonly clientDetails: IClientDetails;
-    readonly storage: IDocumentStorageService;
+    readonly storage: IDocumentStorageService | undefined;
```

Comment:
@jatgarg main shouldn't have undefined correct
```suggestion
    readonly storage: IDocumentStorageService;
```